scripts/environment.sh

```diff
@@ -66,14 +66,6 @@ if [[ ${GINKGO_FOCUS:-} == "upgrade" ]]; then
   export KUBERNETES_VERSION=${FROM_K8S_VERSION}
```

Comment:
I think we don't need these two vars, probably these can be removed now.

Reply:
done